packages/overlay/src/overlay-stack.ts

```diff
@@ -90,8 +90,9 @@ export class OverlayStack {
             return;
         }
         this.document.body.attachShadow({ mode: 'open' });
-        /* c8 ignore next 3 */
+        /* c8 ignore next 4 */
         if (!this.document.body.shadowRoot) {
+            this.trappingInited = false;
```

Comment:
Instead, can we move line 86 to after this block? Seems like a reasonable change, even if short lived.

Reply:
yes makes sense.